adal/src/main/java/com/microsoft/aad/adal/TokenCacheAccessor.java

```diff
@@ -80,33 +86,60 @@ TokenCacheItem getATFromCache(final String resource, final String clientId, fina
         
         return accessTokenItem;
     }
-    
+
     /**
      * @return {@link TokenCacheItem} for regular token cache entry.  
      */
     TokenCacheItem getRegularRefreshTokenCacheItem(final String resource, final String clientId, final String user) {
+        final CacheEvent cacheEvent = startCacheTelemetryRequest(EventStrings.TOKEN_TYPE_RT);
```

Comment:
>final CacheEvent cacheEvent = startCacheTelemetryRequest(EventStrings.TOKEN_TYPE_RT); [](start = 8, length = 85)

what will the telemetry data look like? will it show like RT lookup, RT found/RT not found, RT lookup done?